TrustModel.md

```diff
@@ -0,0 +1,426 @@
+
+- Container launch stage
+  - Measure and attest the TEE where confidential containers run to prove the deployment and initialization of pre-container are valid and expected;
+  - Pull the encrypted and signed container image in TEE;
+  - Provision the runtime configurations through a secure and attested control plane with the protections of confidentiality and integrity.
+
+- Workload operation stage
+  - Provide the protections of confidentiality and integrity for the storage and transport of data derived from TEE outside the TEE instance;
+  - Keep the dynamic measurement of TEE where confidential containers run, and store the measurements to a secure location;
+  - Continuously launch the runtime attestations for TEE where confidential containers run according to the policy to ensure the TEEs are not comprised or out-of-date;
+  - Migration of confidential containers between TEE instances in a secure manner.
+
+### Non-goals
+
+The following itesm are considered out-of-scope for the threat model of confidential container: 
+
+- The assurance of protections for application itself.
+
+- The "Availability" in CIA Triad. A specific solution can provide the protection against availability attacks.
+
+- Items related to any software TEEs.
+
+---
+
+## Confidential Computing Threat Vector Areas
+
+The threat vectors[^1] 5.2.1 In-Scope considered to be in-scope for Confidential Computing are 
+defined within the Confidential Computing Consortium Document. The following threat vectors are 
+relate to confidential containers:
+
+- Software supply chain attacks: include the attacks on components directly associated with the protections that specific TEEs provide, as well as pre-container components running inside TEEs. These components belong to the TCB of confidential container. Any attacks that may compromise TCB can fundamentally break the security of confidential container.
+
+- Container image attacks: include the attacks on the container image. A container image includes the tenant's sensitive workload and data, so any attacks that may compromise the container image may cause the leakage of the tenant's sensitive information.
+
+- Control plane attacks: include the attacks on K8s and container runtime control planes (such as CRI/shim/agent API). Traditionally, tenants trust these control planes controlled by the platform owner. If a specific confidential container based solution distrusts these control planes, the storage and transport of data over these control planes needs the protections for confidentiality and integrity. In addition, the Alternately, use another control plane to protect the data, e.g, a secure and attested control plane established with remote attestation procedure. In addition, the service APIs located at the trust boundary, e.g, agent API used by kata-cc, may cause trust boundary violation, without hardening or restriction.
```

Comment:
Here we say if the control plane is trusted, but maybe we can be more direct here and take a position that CC either trusts or does not trust the control plane. 
(also might want to word wrap, and s/In addition, the Alternately/Alternatively or In addition)

Reply:
in general we don't trust the control plane and I think we should be more clear about that.

Maybe what this is getting at is that there are different ways to go about controlling things when you distrust the control plane. For instance, you can pass encrypted data through the control plane i.e. a wrapped Kubernetes secret. We've also discussed having a separate guest-owner provisioned control plane that is trusted.